YoutubeExplodeTests/YoutubeClientPlaylistParsingTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using YoutubeExplode.Services;
+
+namespace YoutubeExplode.Tests
+{
+    [TestClass]
+    public class YoutubeClientPlaylistParsingTests
```

Comment:
Is there a good reason to use mocks here? The reason I opted for real-life integration tests was because youtube has a tendency to change stuff. Mocked unit tests basically only cover the parsing part, which is very basic to begin with. Integration tests, on the other hand, cover both the parsing and the integration part. They also are easier to maintain because you don't need to keep and update the fake response files.